app/controllers/feedback/post-index-controller.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+// NPM dependencies
+const lodash = require('lodash')
+const logger = require('winston')
+const zendeskClient = require('node-zendesk')
+
+// Local dependencies
+const paths = require('../../paths')
+
+const zendesk = zendeskClient.createClient({
+  username: 'zd-api-pay@digital.cabinet-office.gov.uk',
+  token: process.env.ZENDESK_API_KEY,
+  remoteUri: 'https://govuk.zendesk.com/api/v2'
+})
+
+module.exports = (req, res) => {
+  const message = `Service name: ${req.body['service-name']}
+External ID: ${req.body['service-external-id']}
+Gateway account: ${req.body['service-gateway']}
+Feedback rating: ${req.body['feedback-rating']}
+----
+${req.body['feedback-suggestion']}`
+
+  zendesk.tickets.create({
+    ticket: {
+      requester: {
+        email: req.body.email,
+        name: '(no name supplied)'
+      },
+      type: 'question',
+      subject: `Feedback from: ${req.body['service-name']}`,
+      comment: {
+        body: message
+      },
+      group_id: 20487877,
```

Comment:
perhaps put these somewhere

```
config/zendesk.js
module.exports = {
  GROUP_ID: 20487877,
  ORG_ID: 21891972,
  ASSIGNEE_ID: 360607861465,
}

```

curious as to your thoughts, but I won't block the PR for this as they are only used here.

Reply:
Yeah that probably a good idea. Will change, got some weird test bug I need to sort anyway